internal/boxcli/run.go

```diff
@@ -53,6 +54,18 @@ func listScripts(cmd *cobra.Command, flags runCmdFlags) []string {
 }
 
 func runScriptCmd(cmd *cobra.Command, args []string, flags runCmdFlags) error {
+	if len(args) == 0 {
+		scripts := listScripts(cmd, flags)
+		if len(scripts) == 0 {
+			return redact.Errorf("no scripts defined in devbox.json")
```

Comment:
this will return a non-zero exit code, which I feel isn't what we want: its _okay_ for there to not be any scripts to list :)

Could we change this to a regular `fmt.Fprintln(cmd.OutOrStdout(), "no scripts defined in devbox.json")`?